packages/main/src/AvatarGroup.hbs

```diff
@@ -1,17 +1,17 @@
 <div class="ui5-avatar-group-root">
-	<div 
-		class="ui5-avatar-group-items" 
-		@keyup="{{_onkeyup}}" 
-		@keydown="{{_onkeydown}}" 
-		@focusin="{{_onfocusin}}" 
+	<div
+		class="ui5-avatar-group-items"
+		@keyup="{{_onkeyup}}"
+		@keydown="{{_onkeydown}}"
+		@focusin="{{_onfocusin}}"
 		tabindex="{{_groupTabIndex}}"
+		?data-sap-focus-ref="{{_isGroup}}"
 		@click="{{_onGroupClick}}"
 		@ui5-click="{{_onUI5Click}}"
 	>
 		<slot></slot>
-
-		<ui5-button ?hidden="{{_overflowBtnHidden}}" tabindex="{{_overflowButtonTabIndex}}" class="ui5-avatar-group-overflow-btn">
-			{{_overflowButtonText}}
+		<ui5-button ?hidden="{{_overflowBtnHidden}}" ?non-interactive="{{_isGroup}}" class="ui5-avatar-group-overflow-btn">
+			{{_effectiveText}}
```

Comment:
The old name `_overflowButtonText` is ok, we don't need the `_effective` prefix any more since  there is no public property with the same name in this implementation (contrary to the first one)

Make the function `validateEffectiveText` a getter called `_overflowButtonText` and remove the metadata property